docs/conf.py

```diff
@@ -10,7 +9,6 @@
 import os
 import sys
 
-
```

Comment:
Sanic codebase was roughly 50%/50% one or two lines. I tried both and both made fairly large diffs of this, so I went with Ruff default which is one line. Looks like folders other than sanic (guide, tests etc) used different formatting because black only applied to sanic. Can be easily changed with config option if so desired.

Reply:
Sanic was somewhat 50%/50% one or two lines after imports. The guide and other parts not included in automatic formatting had one line while the main codebase had two. I opted for Ruff default, which is one line, to unify everything and to avoid extra space that might not be wanted in guide.

We can provide different configuration for different parts of Sanic code if so desired. I personally don't care about the specific number of empty lines there (or the various other positions where two lines might be inserted).